shared_model/backend/protobuf/queries/impl/proto_query.cpp

```diff
@@ -150,3 +170,5 @@ namespace shared_model {
 
   }  // namespace proto
 }  // namespace shared_model
+
+#undef IROHA_BIND_TYPE
```

Comment:
I think it would be better to `undef` macro right after its last usage.

Reply:
It's right there. It's pretty to save the scope consistence.

Reply:
Looks better.